lib/spack/docs/getting_started.rst

```diff
@@ -1422,122 +1420,137 @@ Step 1: Install prerequisites
 To use Spack on Windows, you will need the following packages:
 
 * Microsoft Visual Studio
-* Python
+* Intel Fortran
```

Comment:
Some of these packages should be optional (however you want to note that in the documentation): Intel Fortran, Perl, NASM. They won't be needed by all users